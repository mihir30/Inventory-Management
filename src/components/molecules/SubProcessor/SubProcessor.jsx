import React from 'react';

const SubProcessor = () => {
    return (
        <div>            
            <div className='group2'>
                <label className='form-label'>
                    Vendor
                </label>
                <input className='form-control sub-input' type='text' defaultValue="Intel"/>
            </div></div>
    )
}

export default SubProcessor;