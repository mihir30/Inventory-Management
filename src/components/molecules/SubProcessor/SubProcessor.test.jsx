import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import SubProcessor from './SubProcessor';

describe("Sub Processor Component", () => {

    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><SubProcessor></SubProcessor></BrowserRouter>, divEl); 
        
    });

    it('should have the right sub processor component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});