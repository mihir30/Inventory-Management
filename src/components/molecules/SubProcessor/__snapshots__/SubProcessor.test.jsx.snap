// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sub Processor Component should have the right sub processor component snapshot 1`] = `null`;
